src/manifest.json

```diff
@@ -69,6 +69,7 @@
 
   "permissions": [
       "<all_urls>",
+      "identity",
```

Comment:
Question: Will this trigger a permission update? 

Also, it looks like it is, but I want to confirm cross-compat with Chrome: https://developer.chrome.com/docs/extensions/reference/identity/